source/agora/node/BlockStorage.d

```diff
@@ -956,28 +953,15 @@ public class BlockStorage : IBlockStorage
         Read the file data, calculate checksum,
         and write checksum to file at start point
 
-        Returns:
```

Comment:
Does it need a `Throws`?

Reply:
Hum, it doesn't throw directly but is not `nothrow`... I plan to inline it in the future, and I don't think it's that important for private functions.